### PR TITLE
created account page

### PR DESCRIPTION
**Pull request by wolffles**
description: this branch was to create a basic interface to view account information such as test_sets.
test coverage: 98.33%

resolving card #11

### CODE REVIEW

File: spec/controllers/user/user_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/controllers/user/user_controller_spec.rb:1]: @wolffles This says Welcome and it should also have some tests.

File: coverage/.resultset.json [hunk 2]
```diff
@@ -1,17 +1,10 @@
 {
```
Comment (brittanyjspringer) [inline coverage/.resultset.json:1]: @wolffles please add your /coverage/* folder to your .gitignore. I'd prefer not to see these files.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by wolffles** [comment]
i added a test for the controller index action but the redirect helper function I don't know how to test for that. it just adds a parameter to devises sessions controller, so can I just leave that?

**Comment by brittanyjspringer** [comment]
Yes, you can just leave the sessions method since Devise covers it.

**Pull request merged**